src/GeneralStateTestsFiller/stEIP150singleCodeGasPrices/gasCostExpFiller.yml

```diff
@@ -0,0 +1,100 @@
+gasCostExp:
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentGasLimit: 100000000
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+# Human readable information
+  _info:
+    comment: "Ori Pomerantz qbzzt1@gmail.com"
+
+  pre:
+
+    095e7baea6a6c7c4c2dfeb977efac326af552d87:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code: |
+        {
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; Initialization
+ 
+          ; Variables (0x20 byte wide)
+          (def 'powerOf           0x000)  ; A to the power of @powerOf
+          (def 'expectedCost      0x020)  ; Expected gas cost
+          (def 'gasB4             0x040)  ; Before the action being measured
+          (def 'gasAfter          0x060)  ; After the action being measured
+          
+          ; Understand CALLDATA. It is four bytes of function
+          ; selector (irrelevant) followed by 32 byte words
+          ; of the parameters
+          [powerOf]       $4
+          [expectedCost]  $36
+
+ 
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; Run the operation
+          [gasB4]    (gas)
+          (exp 2 @powerOf)
+          [gasAfter] (gas)
```

Comment:
here the cost of (GAS) opcode itself gona be recorded. 
you can put this as a constant in a contract and to the formula.

Reply:
That is the reason what I check is the total expected cost.